CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## ${next-version} - ${release-date}
+
+**Features:**
+
+**Enhancements/Fixes:**
+- An issue was fixed in [#1757](https://github.com/KomodoPlatform/atomicDEX-API/pull/1757) where we don't have to wait for all EVM nodes to sync the latest account nonce.
```

Comment:
We might also ref issue tags in the statements, which might cause confusions wheter if it's PR tag or an issue tag. To avoid such confusion, it would be better to keep the URL at the end of logs.

Reply:
Done